entrypoint.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+# check codecov token exist.
```

Comment:
Please move this whole block down after line 13: `echo "-- Pushing coverage to Codecov..."`